src/main/java/gameframework/drawing/GameUniverseViewPortDefaultImpl.java

```diff
@@ -24,11 +29,16 @@ public GameUniverseViewPortDefaultImpl() {}
 	public GameUniverseViewPortDefaultImpl(GameData data) {
 		setGameData(data);
 	}
-
+	
+	/**
+	 * 
+	 * @return the URl of the background Image of the ViewPort. It should be inherited if you use a child class
```

Comment:
English: /child class/ → subclass. And methods from a superclass are always inherited, what you meant here was /redefined/